internal/rm/tegra_devices.go

```diff
@@ -72,3 +72,8 @@ func (d *tegraDevice) GetNumaNode() (bool, int, error) {
 func (d *tegraDevice) GetTotalMemory() (uint64, error) {
 	return 0, nil
 }
+
+// GetComputeCapability is unimplemented for a Tegra device.
+func (d *tegraDevice) GetComputeCapability() (string, error) {
+	return "v0.0", nil
+}
```

Comment:
They do. I just didn't implement it here since we handle them differently from the perspective of the device plugin. Since MPS is not supported on Tegra-based systems, would you be OK with a TODO here?

Reply:
Ack